src/assets/custom/css/_sass/_soft-mod-cheat-sheet-cta-banner.scss

```diff
@@ -0,0 +1,36 @@
+.soft-mod-cheat-sheet-cta-block-image-en {
```

Comment:
The classnames in this file seem inconsistent. The file name, the related HTML file and the different classnames, I'd like to see these using the kind of consistency we planned for `section-intro`